src/entities/coach/behavior/gotolookto/behavior_gotolookto.cpp

```diff
@@ -0,0 +1,64 @@
+/***
+ * Maracatronics Robotics
+ * Federal University of Pernambuco (UFPE) at Recife
+ * http://www.maracatronics.com/
+ *
+ * This file is part of Armorial project.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ ***/
+
+#include "behavior_gotolookto.h"
+
+Behavior_GoToLookTo::Behavior_GoToLookTo() {
+    _positionToGo = Position(false, 0.0, 0.0);
+    _positionToLook = Position(false, 0.0, 0.0);
+}
+
+QString Behavior_GoToLookTo::name() {
+    return "Behavior_GoToLookTo";
+}
+
+void Behavior_GoToLookTo::setPositionToGo(Position &positionToGo) {
+    _positionToGo = positionToGo;
+}
+
+void Behavior_GoToLookTo::setPositionToLook(Position &positionToLook) {
+    _positionToLook = positionToLook;
+}
+
+void Behavior_GoToLookTo::configure() {
+    // Starting skills
+    _skill_goTo = new Skill_GoTo();
```

Comment:
In the configure method, you should create the pointers and use **addSkill(skillPtr*)** method to add them to the behavior!